output/cloud/expv2/collect.go

```diff
@@ -2,12 +2,24 @@ package expv2
 
 import (
 	"errors"
+	"strconv"
 	"sync"
 	"time"
 
+	"go.k6.io/k6/cloudapi/insights"
+	"go.k6.io/k6/js/modules/k6/experimental/tracing"
+	"go.k6.io/k6/lib/netext/httpext"
 	"go.k6.io/k6/metrics"
 )
 
+const (
+	scenarioTag = "scenario"
+	groupTag    = "group"
+	nameTag     = "name"
```

Comment:
Is this `name` tag by default set to `http_url` if users don't specify a value for it? My current understanding is that this tag is used to deal with cardinality issues ([source](https://k6.io/docs/using-k6/http-requests/#url-grouping)).

Reply:
It is set by default if it is enabled - which it must be for the usage of the cloud output. 

Other required tags for tis output include `method`, `status`, `error`, `check` and `group`

https://github.com/grafana/k6/blob/3ab0085ded5b85b54b7147faaa85e1520ecff7b4/output/cloud/output.go#L134-L154

Reply:
What about the `scenario` tag?